icu4c/source/data/misc/supplementalData.txt

```diff
@@ -7860,6 +7860,7 @@ supplementalData:table(nofallback){
                 "Qaaj~p",
             }
             special{
+                "Aran",
```

Comment:
Let's keep this branch clean; don't check in changes unrelated to units

Reply:
Done.